java-core/google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -767,4 +791,48 @@ public String getClientLibToken() {
   public String getQuotaProjectId() {
     return quotaProjectId;
   }
+
+  /**
+   * Returns the resolved endpoint for the Service to connect to Google Cloud
```

Comment:
Would you add a paragraph that explains the specification of the resolution in this Javadoc?